packages/libglib/Cargo.toml

```diff
@@ -12,8 +12,8 @@ path = "../packages.rs"
 releases-url = "https://download.gnome.org/sources/glib"
 
 [[package.metadata.build-package.external-files]]
-url = "https://download.gnome.org/sources/glib/2.83/glib-2.83.0.tar.xz"
-sha512 = "293a164d93441e89303f65fda7ce1d212040e5ffbe526d121c2f0f352a9074235ae687430889b8a56d003e067931bf70ad69653cdafb0dcb64f6dfa683177375"
+url = "https://download.gnome.org/sources/glib/2.84/glib-2.84.0.tar.xz"
```

Comment:
Can you add a comment above the releases URL that glib uses the "stable releases use even numbers" versioning scheme? So we don't move off of stable releases again.